src/device-registry/validators/uptime.validators.js

```
@@ -10,183 +11,195 @@
-    .custom((value) => {
-      if (!moment(value, moment.ISO_8601, true).isValid()) {
-        throw new Error(
-          `Please provide a valid ISO formatted datetime string (YYYY-MM-DDTHH:mm:ss.sssZ)`
-        );
+const uptimeValidations = {
+  pagination: (defaultLimit = 1000, maxLimit = 2000) => {
+    // Add pagination here
+    return (req, res, next) => {
+      let limit = parseInt(req.query.limit, 10);
+      const skip = parseInt(req.query.skip, 10);
+      if (isNaN(limit) || limit < 1) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)